pkg/pgmodel/migrations/sql/1_base_schema.up.sql

```diff
@@ -797,7 +817,7 @@ BEGIN
 
    EXECUTE format($query$
     WITH CTE AS (
-        SELECT SCHEMA_PROM.label_array($1)
+        SELECT SCHEMA_PROM.get_or_create_label_array($1)
```

Comment:
should the parent function here `SCHEMA_PROM.series_id(` be renamed and go into SCHEMA_CATALOG?

Reply:
it's a GoC right?